tasks/util/constants.js

```diff
@@ -84,16 +84,29 @@ module.exports = {
     testContainerHome: '/var/www/streambed/image_server/plotly.js',
 
     uglifyOptions: {
-        fromString: true,
         mangle: true,
         compress: {
-            warnings: false,
-            screw_ie8: true
+            warnings: false
         },
         output: {
             beautify: false,
             ascii_only: true
-        }
+        },
+        sourceMap: false
+    },
+
+    bubleifyOptions: {
+        target: {
```

Comment:
Hmm. We have to target browsers waaaaaaaay older than that. Can bubleify guarantee to output code that IE11 understands?

Reply:
@etpinard that is the best bubleify can do, it is targeted only for ES6 → ES5, not ES5 → ES3 etc.

Reply:
Hmm. Well that won't work then. plotly.js needs to be pure ES5.

Reply:
@etpinard that is pure ES5, same as what we have now. `target` flag only indicates what new ES6 features are supported by target environment, eg. some browsers have `Object.assign` etc, so since we indicate oldest target possible, we make sure we get pure ES5.

Reply:
Could we put these config options in the package.json instead, so that all consumers bundling with browserify or webpack+ify-loader get the same results as us in our `dist/`?

Reply:
Can the output bundle be minified by plain-old uglify-js? e.g. does

```
browserify -t bubleify src/plotly.js | uglify-js > bundle.js
```

work?

Reply:
I'm not a big fan of this `target` option. I find it pretty confusing. Could we instead manually list all the [transforms](https://buble.surge.sh/guide/#command-line-options) we want turned on?